chainstate/tx-verifier/src/transaction_verifier/timelock_check.rs

```diff
@@ -139,5 +138,38 @@ pub fn check_timelocks<
         }
     }
 
+    if let Some(outputs) = tx.outputs() {
```

Comment:
Didn't we discuss this and agree that this should be done in check_block because the history is not required?

Reply:
right, sorry totally forgot that

Reply:
ok so I moved it to check_block, but it requires me to provide the function with current block height. Because maturity setting is taken from netupgrade

Reply:
That's OK.